src/config.py

```diff
@@ -7,4 +7,10 @@
 TOKEN = os.environ.get("TOKEN")  # required
 DEBUG = os.environ.get("DEBUG", False)
 
-CHANNEL_IDS = {"lobby": 627542044390457350}
+ROLE_IDS = [
+  ("Webgroup", "1166751688598761583"),
+  ("Gamez", "1089204642241581139"),
+  ("Croomer", "1172696659097047050"),

```

Comment:
Might want to use 4 spaces for indentation here, for consistency.
Could also use a tuple instead of a list, as it should be immutable.